pycloudlib/lxd/cloud.py

```diff
@@ -437,6 +437,10 @@ def clean(self) -> List[Exception]:
                     exceptions.append(e)
         return exceptions
 
+    def version(self):
```

Comment:
Same, I'd cache this value somehow as it's unlikely to change on the host mid testing.